exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0) {
+    return "negative";
+  } else if (number >= 0) {
```

Comment:
You don't need a second if condition here.  If you remove the (number >= 0) this function will still work.  If the number is 0 or less it's "negative' else it's "positive".

Reply:
> You don't need a second if condition here. If you remove the (number >= 0) this function will still work. If the number is 0 or less it's "negative' else it's "positive".

Corrected. Thank you